framework/tlvf/src/include/tlvf/CmduParser.h

```diff
@@ -0,0 +1,35 @@
+/* SPDX-License-Identifier: BSD-2-Clause-Patent
+ *
+ * Copyright (c) 2016-2019 Intel Corporation
+ *
+ * This code is subject to the terms of the BSD+Patent license.
+ * See LICENSE file for more details.
+ */
+#ifndef _CmduParser_H_
+#define _CmduParser_H_
+
+#include <memory>
+#include <BaseClass.h>
+
+namespace ieee1905_1 {
+    class CmduParser
```

Comment:
Formatting is all wrong, please use clang-format on it (clang-format.sh skips it so it has to be done manually).